tests/deployment_utils.py

```diff
@@ -223,10 +223,16 @@ def _create_temp_config(self, config: str):
     def run(self, config: str, force_installed: bool = False, vcs_type: str = "none",
             additional_parameters="", environment=None, expected_to_fail=False, workdir=None):
 
-        # workdir is ignored unless force_installed;
-        # if installed to make sure local sources are not used by default the directory is changed
+        # as nonci changes workdir to project root, Universum is always an external moudule
```

Comment:
The connection these two statements is not clear to me:
1. nonci changes the workdir to project root
2. Universum is an external module

nonci is one part of the Universum. So, the respective nonci implementation is chosen depending on how the Universum is launched. The change of the workdir happens only after that.

Probably, I just misunderstand the comment.